File: app/backend/models/tipo_expediente_model.py
from sqlalchemy import Column, Integer, String, Text, Boolean
from backend.database.connection import Base




class TipoExpediente(Base):
    __tablename__ = "TipoExpediente"

    IdTipoExpediente = Column(Integer, primary_key=True, index=True, nullable=False)
    Nombre = Column(String(100), nullable=False)
    Descripcion = Column(Text, nullable=True)
    Activo = Column(Boolean, nullable=True)

File: app/frontend/src/app/features/tipos-expediente/models/tipo-expediente.model.ts
export interface TipoExpediente {
  IdTipoExpediente: number;
  Nombre: string;
  Descripcion: string;
  Activo: boolean;
  AudFecha?: string;
  AudUsuario?: number;
}

export interface TipoExpedienteCreate {
  Nombre: string;
  Descripcion: string;
  Activo: boolean;
}

export interface TipoExpedienteUpdate {
  Nombre: string;
  Descripcion: string;
  Activo: boolean;
}